app/i18n/en/feedback.php

```diff
@@ -36,6 +36,7 @@
 	),
 	'extensions' => array(
 		'already_enabled' => '%s is already enabled',
+		'cannot_delete' => '%s cannot be deleted',
```

Comment:
I left the comment only on one line but I did mean it for both. ;-)

Reply:
What do you have in mind for that one?